examples/src/main/res/layout/activity_instruction_view_layout.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <com.mapbox.mapboxsdk.maps.MapView
+        android:id="@+id/mapView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:mapbox_uiAttribution="false"
+        app:mapbox_uiCompass="true"
+        app:mapbox_uiCompassGravity="bottom|left"
+        app:mapbox_uiCompassMarginBottom="110dp"
+        app:mapbox_uiCompassMarginLeft="16dp"
+        app:mapbox_uiLogo="false" />
+
+
+    <ImageView
+        android:id="@+id/screenshotView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:visibility="invisible"
+        tools:ignore="ContentDescription" />
```

Comment:
Same as above

Reply:
same as?

Reply:
Same explaination as above on why we should not use `match_parent`